frontend/src/app/modules/bim/ifc_models/xeokit/xeokit-server.js

```diff
@@ -71,8 +71,8 @@ class XeokitServer {
   }
 
   attachmentUrl(attachmentId) {
-    return "/api/v3/attachments/" + attachmentId + "/content";
+    return window.appBasePath + "/api/v3/attachments/" + attachmentId + "/content";
```

Comment:
Ideally this would just be taken from the path helper service but someone with more frontend knowledge ought to do that since I don't know how. For the time being this is the fix for the immediate issue, though.